op-guide/security.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 开启TLS验证
+category: deployment
+---
+
+# 开启TLS验证
+
+## 概述
+
+本文档介绍 TiDB 集群如何开启 TLS 验证，其支持
+
+- TiDB组件之间的双向验证，包括 TiDB、TiKV、PD 相互之间， TiKV Control 与 TiKV、PD Control 与 PD 的双向认证，以及 TiKV peer 之间、 PD peer 之间。一旦开启，所有组件之间均使用验证，不支持只开启某一部分的验证。
```

Comment:
- Add a space: after the first "TiDB".
- Delete the extra space: 
    - before "TiKV Control"
    - before "PD peer"